src/components/GalleryPage/GalleryMenu.js

```diff
@@ -16,9 +16,17 @@ function GalleryMenu({ hothNames, galleryLinks }) {
 	const handleClose = () => {
 		setAnchorEl(null);
 	};
+	const linkStyles = {
```

Comment:
You're right that there are better ways to add styling. For styling, we usually use material-ui's API to do styling. For example, take a look at `src/components/SchedulePage/Schedule.js` and how they use `makeStyles` and `useStyles` to do styling for an entire component. This also lets us access the `theme` object that we need for colors.